spec/index.bs

```diff
@@ -0,0 +1,404 @@
+<pre class='metadata'>
```

Comment:
Nice job! The bikeshed version looks much nicer and I like the simplification of the algorithms.